src/model/Note.ts

```diff
@@ -0,0 +1,4 @@
+export default interface Note {
+  noteDate: Date
```

Comment:
I think we should use `date` instead of `noteDate` since name of the interface is `Note`